consensus/src/round_manager.rs

```diff
@@ -1037,6 +1186,40 @@ impl RoundManager {
         result
     }
 
+    // Insert ordered certificate formed by aggregating order votes
+    async fn new_ordered_cert(
```

Comment:
nit: `process_ordered_cert` ?

Reply:
Currently, we have `new_qc_aggregated` and `new_2chain_tc_aggregated` in round manager. Just following the naming convention along the same lines.